source/_components/lightwave.markdown

```diff
@@ -2,7 +2,7 @@
 layout: page
 title: "Lightwave"
 description: "Instructions on how to integrate Lightwave devices with Home Assistant."
-date: 2018-12-03 18:00
+date: 2019-02-11 19:00
```

Comment:
It is not the intention that this date is changed every time there is a change.

Reply:
OK - I will resolve these changes this evening.